### ISSUE TITLE
Copy nodes as ASCII Text

### ISSUE THREAD

**Issue by 70RMUND** [ReClassNET/ReClass.NET#99]
As an enhancement request it may be useful to highlight a span of nodes and copy its representative ASCII output (as shown in UI) directly into clipboard. This would be useful for analyzing multiple class instances in a diff tool to see which offsets pop out as fields that differ.

**Comment by KN4CK3R** [comment]
Good idea but a cooler enhancement would be two panels showing the same class side by side. The panels could highlight different values for both instances. Currently that's not possible because the internal data modell can't handle this but it would be a good addition.
